### PR TITLE
Add specs for string split

### LINKED ISSUE #1832: Add tests for string.split()
See https://github.com/sass/sass/issues/1950

### PR DESCRIPTION
**Pull request by dvdherron**
See https://github.com/sass/sass/issues/1950
Closes: https://github.com/sass/sass-spec/issues/1832

See the latest version of the proposal [here](https://github.com/sass/sass/blob/main/accepted/string-split.md#split)

[skip dart-sass]

### CODE REVIEW

**Comment by dvdherron** [comment]
@mirisuzanne @stacyk 
Mainly looking for feedback on whether or not it seems like I'm missing any tests. Took a guess at file naming. If you have thoughts on how that could possibly be improved before I send this to Sass that would be great too. Thanks!

File: spec/core_functions/string/split.hrx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+<===> double_width_character/output.css
+a {
+  b: ["👭", "a"];
+}
+
+<===>
+================================================================================
+<===> private_use_character/input.scss
+// Dart Sass emits private-use characters as escapes in expanded mode, but it
+// should stil treat them as single characters for the purpose of functions.
```
Comment (stacyk) [inline spec/core_functions/string/split.hrx]: typo: still

**Review by mirisuzanne — APPROVED** [review]
looks good to me

File: spec/core_functions/string/split.hrx (hunk 1 above)
Comment (dvdherron) [reply spec/core_functions/string/split.hrx]: Good catch. I'll fix.

**Pull request closed without merging**